### PR TITLE
Adding configuration files to bootstrap the VM

### LINKED ISSUE #1: Add Vagrantfile and setup script
In order for all the contributors to have a nice and simple way to work on the same encapsulated env we need to ad vagrant to the project and some scripts to automatically run the project with one command

### PR DESCRIPTION
**Pull request by Gal-f**
Adding configuration files in order for the team to work in a uniform environment.

- Added Vagrantfile to setup VM box
- Modified .gitignore to include Vagrant and Django files to be ignored
- Added Pipfile and Pipfile.lock to define installation requirements on the VM
- Added bootstrap.sh to run the entire bootstrap automatically

This resolves #1

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  # Provider-specific configuration so you can fine-tune various
+  # backing providers for Vagrant. These expose provider-specific options.
+  # Example for VirtualBox:
+  #
+  config.vm.provider "virtualbox" do |vb|
+  #   # Display the VirtualBox GUI when booting the machine
+  #   vb.gui = true
+  #
+  #   # Customize the amount of memory on the VM:
+  vb.memory = "1024"
```
Comment (ifireball) [inline Vagrantfile:58]: generally, when code is in a do...end  block it should be indented to make it more readable. The Vagrantfile is written in ruby so its not critical, but is a good idea to maintain indention anyway

**Review by ifireball — APPROVED** [review]
I would generally expect the Pipenv setup to be a part of the Django task, but it can be a part of the Vagrant task as well.

**Comment by ifireball** [comment]
Let me know if you want to make any more adjustments and corrections to this Pr, if not, we can merge it.

**Comment by Gal-f** [comment]
> Let me know if you want to make any more adjustments and corrections to this Pr, if not, we can merge it.

Merge it please. 
Everything seems OK and we're on a schedule.
Thanks

**Pull request merged**